RELEASES.md

```diff
@@ -39,46 +39,41 @@ be done from that branch. For example, once we release `v1.0.0`, a branch
+releases which are already scheduled.
 
-While pre-releases are done to assist in the stabilization process, no
-guarantees are provided.
+The maintainers will maintain a roadmap and milestones for each release, however,
+features may be pushed to accomodate the release timeline. If your issue or feature
+is not present in the roadmap, please open a Github issue or leave a
+comment requesting it be added to a milestone.
 
-### Upgrade Path
+### Patch Releases
 
-The upgrade path for containerd is such that the 0.0.x patch releases are
-always backward compatible with its major and minor version. Minor (0.x.0)
-version will always be compatible with the previous minor release. i.e. 1.2.0
-is backwards compatible with 1.1.0 and 1.1.0 is compatible with 1.0.0. There is
-no compatibility guarantees for upgrades that span multiple, _minor_ releases.
-For example, 1.0.0 to 1.2.0 is not supported. One should first upgrade to 1.1,
-then 1.2.
-
-There are no compatibility guarantees with upgrades to _major_ versions. For
-example, upgrading from 1.0.0 to 2.0.0 may require resources to migrated or
-integrations to change. Each major version will be supported for at least 1
-year with bug fixes and security patches.
+Patch releases are made directly from release branches and will be done as needed
+by the release branch owners.
 
-### Next Release
+### Pre-releases
 
-The activity for the next release will be tracked in the
-[milestones](https://github.com/containerd/containerd/milestones). If your
-issue or PR is not present in a milestone, please reach out to the maintainers
-to create the milestone or add an issue or PR to an existing milestone.
+Pre-releases, such as alphas, betas and release candidates will be conducted
+from their source branch. For major and minor releases, these releases will be
+done from main. For patch releases, it is uncommon to have pre-releases but
+they may have an rc based on the discretion of the release branch owners.
 
 ### Support Horizon
 
 Support horizons will be defined corresponding to a release branch, identified
 by `<major>.<minor>`. Release branches will be in one of several states:
 
-- __*Next*__: The next planned release branch.
+- __*Future*__: An upcoming scheduled release.
+- __*Alpha*__: The next scheduled release on the main branch under active development.
+- __*Beta*__: The next scheduled release on the main branch under testing. Begins 8-10 weeks before a final release.
+- __*RC*__: The next scheduled release on the main branch under final testing and stabilization. Begins 2-4 weeks before a final release. For new releases where the source branch is main, the main branch will be in a feature freeze during this phase.
 - __*Active*__: The release is a stable branch which is currently supported and accepting patches.
-- __*Extended*__: The release branch is only accepting security patches.
+- __*Extended*__: The release branch is only accepting security patches and critical updates.
```

Comment:
What constitutes a critical update? (or, in other words: what differentiates Extended from Active or LTS?)

Reply:
I'm just going to remove this. There was a proposal to say there could be exceptions based on discretion of the owners. However, we can always assume that is true, it is up to the humans to make a judgement call and the default will be no if its not a security patch.